packages/workbench/src/app/modules/login/login.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="login-form" id="wb-login">
+  <div class="card">
+    <div class="card-body">
+      <h1>{{ 'login.title' | transloco }}</h1>
+
+      <form *ngIf="isGDBLoginEnabled()" [formGroup]="loginForm" (ngSubmit)="login()" novalidate>
```

Comment:
You can use the new Angular view flow control for easy reading.

https://v17.angular.io/guide/control_flow

Here for exaple you can have
```html
@if (isGDBLoginEnabled()) {
...
}
@else {
...
}